docs/GetPackages/index.ts

```diff
@@ -0,0 +1,126 @@
+
+const getLocalPackageSlugs = (): Task<GlobError, string[]> =>
+  getLocalPackageDirs().map(a => a.map(slugFromPackageDir));
+
+const localPackageMetadataFromSlug = (slug: string): Task<ReadFileError, PackageMetadata> =>
+  readFileT(`${localPackageDirFromSlug(slug)}/package.json`)
+    .map(packageMetadataDecoder.toJsonFn())
+    .map(requireResultDuringBuild);
+
+const localReadmeFromSlug = (slug: string): Task<ReadFileError, string> =>
+  readFileT(`${localPackageDirFromSlug(slug)}/README.md`);
+
+const localPackageDataFromSlug = (slug: string): Task<ReadFileError, PackageData> =>
+  Task.succeed<ReadFileError, { slug: string }>({ slug })
+    .assign('metadata', localPackageMetadataFromSlug(slug))
+    .assign('readme', localReadmeFromSlug(slug));
+
+type GetLocalPackagesError = GlobError | ReadFileError;
+
+const getAllLocalPackageData = (): Task<GetLocalPackagesError, PackageData[]> =>
+  getLocalPackageSlugs()
+    .mapError<GetLocalPackagesError>(identity)
+    .andThen(a => Task.all(a.map(localPackageDataFromSlug)));
+
+interface FestivePossumPackagePaths {
+  slug: string;
+  readme: GithubFestivePossumPath;
+  packageJson: GithubFestivePossumPath;
+}
+
+const festivePossumPackage = (slug: string, readmeName: string): FestivePossumPackagePaths => ({
+  slug,
+  readme: `${festivePossumPackagesPath}/${slug}/${readmeName}`,
+  packageJson: `${festivePossumPackagesPath}/${slug}/package.json`,
+});
+
+const festivePossumReadmes: Map<string, string> = new Map([
+  ['ajaxian', 'readme.md'],
+  ['ajaxios', 'readme.md'],
+  ['cute', 'README.md'],
+  ['gaia', 'readme.md'],
+  ['jsonous', 'readme.md'],
+  ['maybeasy', 'readme.md'],
+  ['nonempty-list', 'README.md'],
+  ['piper', 'README.md'],
+  ['resulty', 'readme.md'],
+  ['stack-ts', 'README.md'],
+  ['taskarian', 'readme.md'],
+]);
+
+const festivePossumPaths = (): Array<FestivePossumPackagePaths> => {
+  const paths: Array<FestivePossumPackagePaths> = [];
+  festivePossumReadmes.forEach((readme, slug) => paths.push(festivePossumPackage(slug, readme)));
+  return paths;
+};
+
+const festivePossumPackageMetadata = (
+  paths: FestivePossumPackagePaths
+): Task<HttpError, PackageMetadata> =>
+  toHttpTask(get(paths.packageJson).withDecoder(packageMetadataDecoder));
+
+const festivePossumPackageReadmeContent = (
+  paths: FestivePossumPackagePaths
+): Task<HttpError, string> => toHttpTask(get(paths.readme).withDecoder(string));
+
+const festivePossumPackageData = (paths: FestivePossumPackagePaths): Task<HttpError, PackageData> =>
+  Task.succeed<HttpError, { slug: string }>({ slug: paths.slug })
+    .assign('metadata', festivePossumPackageMetadata(paths))
+    .assign('readme', festivePossumPackageReadmeContent(paths));
+
+type GetFestivePossumPackagesError = HttpError;
+
+const getAllFestivePossumPackageData = (): Task<
+  GetFestivePossumPackagesError,
+  Array<PackageData>
+> => Task.all(festivePossumPaths().map(festivePossumPackageData));
```

Comment:
I updated this to use `Task.all`, which runs the tasks in parallel, and the dev experience is much better. Before, the page took several seconds to load while fetching all the packages. Now it takes about 1s for the first page load.